dart/tool/service.md

```diff
@@ -2071,6 +2076,11 @@ class Isolate extends Response {
   // isolate.
   string id;
 
+  // Provided and set to true if the id of an Object is fixed. If true, the id
+  // of an Object is guaranteed not to change or expire. The object may, however,
+  // still be _Collected_.
+  bool fixedId [optional];
```

Comment:
@bkonyi, here (and for IsolateRef), I think we need to add explicit `fixedId`s to the spec.